src/app/home.component.ts

```diff
@@ -46,11 +46,13 @@ export class HomeComponent {
       children: [
         {
           name: 'Sidebar Link is super long and even longer foo bar baz',
-          path: '/0'
+          path: '/0',
+          showInNav: true
         },
         {
-          name: 'Sidebar Link',
-          path: '/1'
+          name: 'Hidden Sidebar Link',
+          path: '/1',
+          showInNav: false
```

Comment:
Don't think these changes are needed any longer.  Custom sidebar routes can achieve this by simply not adding the route to begin with.